assets/i18n/en.json

```diff
@@ -384,7 +384,7 @@
     "I18N_PLAYER_IS_PRIVATE": "This exploration is private.",
     "I18N_PLAYER_LAST_UPDATED_TOOLTIP": "Last Updated",
     "I18N_PLAYER_LEARN_AGAIN_BUTTON": "Learn Again",
-    "I18N_PLAYER_LEAVE_FEEDBACK": "Leave feedback for the authors...",
+    "I18N_PLAYER_LEAVE_FEEDBACK": "Leave feedback for the authors (When submitted, this will also include a reference to the card you're currently at in the exploration.)...",
```

Comment:
Please be more careful with the punctuation. It should be:

"Leave feedback for the authors. (When submitted, this will also include a reference to the card you're currently at in the exploration.)"

(see the original comment).

Reply:
I wasn't sure about whether to remove/place the three dots in the sentence. Thanks for the suggestion.

Done!